### ISSUE TITLE
[Bug] PS5 gyro not working

### ISSUE THREAD

**Issue by xander-haj** [Ryujinx/Ryujinx#5156]
### Description of the issue

ps5 gyro not working, saw another issue mentioned something about steam, steam is not running on my end
video:
https://discord.com/channels/410208534861447168/410208610455519243/1112893612095713310

### Reproduction steps

open input setting with ps5 controller connected via usb, make sure motion setting is enabled 

### Log file

00:00:00.050 |I| Configuration LogValueChange: ResScale set to: 1
00:00:00.054 |I| Configuration LogValueChange: ResScaleCustom set to: 1
00:00:00.054 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.055 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
00:00:00.056 |I| Configuration LogValueChange: BackendThreading set to: Auto
00:00:00.057 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
00:00:00.057 |I| Configuration LogValueChange: PreferredGpu set to: 0x10DE_0x1B81
00:00:00.058 |I| Configuration LogValueChange: AntiAliasing set to: SmaaHigh
00:00:00.058 |I| Configuration LogValueChange: ScalingFilter set to: Bilinear
00:00:00.059 |I| Configuration LogValueChange: ScalingFilterLevel set to: 80
00:00:00.060 |I| Configuration LogValueChange: EnableDockedMode set to: True
00:00:00.061 |I| Configuration LogValueChange: EnableVsync set to: True
00:00:00.061 |I| Configuration LogValueChange: EnableShaderCache set to: True
00:00:00.061 |I| Configuration LogValueChange: EnableTextureRecompression set to: False
00:00:00.061 |I| Configuration LogValueChange: EnableMacroHLE set to: True
00:00:00.061 |I| Configuration LogValueChange: EnablePtc set to: True
00:00:00.061 |I| Configuration LogValueChange: EnableInternetAccess set to: False
00:00:00.061 |I| Configuration LogValueChange: EnableFsIntegrityChecks set to: True
00:00:00.062 |I| Configuration LogValueChange: FsGlobalAccessLogMode set to: 0
00:00:00.062 |I| Configuration LogValueChange: AudioBackend set to: SDL2
00:00:00.062 |I| Configuration LogValueChange: AudioVolume set to: 1
00:00:00.063 |I| Configuration LogValueChange: MemoryManagerMode set to: HostMappedUnsafe
00:00:00.063 |I| Configuration LogValueChange: ExpandRam set to: False
00:00:00.063 |I| Configuration LogValueChange: IgnoreMissingServices set to: False
00:00:00.063 |I| Configuration LogValueChange: UseHypervisor set to: True
00:00:00.067 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.842
00:00:00.072 |N| Application Print: Operating System: Microsoft Windows 10.0.19045 (X64)
00:00:00.072 |N| Application Print: CPU: Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz ; 8 logical
00:00:00.073 |N| Application Print: RAM: Total 32703 MiB ; Available 26499 MiB
00:00:00.076 |N| Application PrintSystemInfo: Logs Enabled: Info, Warning, Error, Guest, Stub
00:00:00.076 |N| Application PrintSystemInfo: Launch Mode: UserProfile
00:00:02.225 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.231 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.235 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.240 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.247 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.251 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.256 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.263 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.268 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.273 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.277 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.283 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.287 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.292 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.301 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.307 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.312 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.317 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.322 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.329 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.335 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.340 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.345 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.351 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.357 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.362 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.367 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.373 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.383 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.388 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.394 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.400 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.405 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.410 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.416 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.421 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.426 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.432 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.439 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.445 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.449 |W| GUI.ApplicationLibraryThread Application LoadApplications: The header key is incorrect or missing and therefore the NCA header content type check has failed. Errored File: G:\Games Switch\Swi\NSP\Lost Ember 0100E3400F242000.nsp
00:00:02.454 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.460 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.466 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.489 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.495 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.507 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.513 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.521 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.527 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.533 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.543 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.550 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.555 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.562 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.567 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.573 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.584 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.593 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.602 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.622 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.627 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.633 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.647 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.654 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.660 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.672 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.679 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.688 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.694 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.706 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.711 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.717 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.722 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.727 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.733 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.738 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.743 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.771 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.777 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.783 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.790 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.822 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:02.838 |W| GUI.ApplicationLibraryThread Application LoadApplications: The header key is incorrect or missing and therefore the NCA header content type check has failed. Errored File: G:\Games Switch\Swi\update\Fire Emblem 3H\010055D009F78000[game].nsp
00:00:02.852 |W| GUI.ApplicationLibraryThread Application LoadApplications: Last played string "Never" is invalid for current system culture, skipping (did current culture change?)
00:00:00.127 |I| Gpu : Backend Threading (Auto): True
00:00:00.447 |N| Application LoadApplication: Using Firmware Version: 16.0.2
00:00:00.447 |I| Application LoadApplication: Loading as XCI.
00:00:00.484 |I| ModLoader QueryContentsDir: Searching mods for Title 0100F2C0115B6000
00:00:00.487 |I| ModLoader AddModsFromDirectory: Found mod '1080p Disable FSR FXAA v1.1.2' [E]
00:00:00.487 |I| ModLoader AddModsFromDirectory: Found mod '1korok_expand' [R]
00:00:00.487 |I| ModLoader AddModsFromDirectory: Found mod '30fps_stable_static_totk' [E]
00:00:00.487 |I| ModLoader AddModsFromDirectory: Found mod 'armor 4 me' [R]
00:00:00.487 |I| ModLoader AddModsFromDirectory: Found mod 'Dualsense UI Mod v11.5 - Western - White' [R]
00:00:00.487 |I| ModLoader AddModsFromDirectory: Found mod 'no-frict-shild-v-1-2' [R]
00:00:00.487 |I| ModLoader AddModsFromDirectory: Found mod 'vah2default' [R]
00:00:00.487 |I| ModLoader AddModsFromDirectory: Found mod 'YouDied' [R]
00:00:00.488 |I| ModLoader AddModsFromDirectory: Found mod 'zonai-addon-inft-rokit' [R]
00:00:00.489 |I| ModLoader QueryContentsDir: Searching mods for Title 0100F2C0115B6000
00:00:00.493 |I| Loader Load: Loading rtld...
00:00:00.504 |I| Loader PrintRoSectionInfo: rtld:
    Module: nnrtld
00:00:00.504 |I| Loader Load: Loading main...
00:00:01.361 |I| Loader PrintRoSectionInfo: main:
    Module: D:\home\Project\EX-King\App\Rom\NX64\Product_Optimize\code\EX-King.nss
    SDK Libraries: SDK MW+Nintendo+NintendoWare_Vfx2-15_3_1-Release
                   SDK MW+Nintendo+NintendoWare_Font-15_3_1-Release
                   SDK MW+Nintendo+NintendoWare_Ui2d-15_3_1-Release
                   SDK MW+Nintendo+NintendoWare_G3d2-15_3_1-Release
                   SDK MW+Nintendo+NintendoWare_Meshoptimizer_For_MeshCodec-0_15_0-Release
                   SDK MW+Microsoft+Havok_Cloth-2022_1
                   SDK MW+Microsoft+Havok_Animation-2022_1
                   SDK MW+Microsoft+Havok_AI-2022_1
00:00:01.361 |I| Loader Load: Loading subsdk0...
00:00:01.422 |I| Loader PrintRoSectionInfo: subsdk0:
    Module: multimedia
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_movie-15_3_1-Release
00:00:01.422 |I| Loader Load: Loading sdk...
00:00:01.554 |I| Loader PrintRoSectionInfo: sdk:
    Module: nnSdk
    FS SDK Version: 15.3.1
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_libz-15_3_1-Release
                   SDK MW+Nintendo+NintendoSdk_nnSdk-15_3_1-Release
                   SDK MW+Nintendo+NintendoSdk_BuildTarget-64Bit
                   SDK MW+Nintendo+NintendoSDK_NVN-15_3_1-Release
00:00:01.564 |I| ModLoader ApplyProgramPatches: Matching IPSwitch patch '1.1.2.pchtxt' in '1080p Disable FSR FXAA v1.1.2' bid=9A10ED9435C06733DA597D8094D9000AB5D3EE6
00:00:01.565 |I| ModLoader Parse: IPSwitch:    # The Legend of Zelda: Tears of the Kingdom [0100F2C0115B6000] v1.1.2 - 1080p Disable FSR FXAA
00:00:01.566 |I| ModLoader Parse: IPSwitch:    print_values 0xcdd68c <= 15000014
00:00:01.568 |I| ModLoader Parse: IPSwitch:    print_values 0xc770e0 <= 08008052
00:00:01.568 |I| ModLoader Parse: IPSwitch:    print_values 0xc770f0 <= 08008052
00:00:01.571 |I| ModLoader ApplyProgramPatches: Matching IPSwitch patch '1.1.2.pchtxt' in '30fps_stable_static_totk' bid=9A10ED9435C06733DA597D8094D9000AB5D3EE6
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0xeac470 <= 41008052
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0x8f68a4 <= 55008052
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0x8f6914 <= 48008052
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0x8f68ac <= 4C008052
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0x8f67f8 <= EB031F2A
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0x19b2084 <= 1F2003D5
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0x19bdae8 <= 1F2003D5
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0x8f67fc <= 55008052
00:00:01.571 |I| ModLoader Parse: IPSwitch:    print_values 0x8f6830 <= 41008052
00:00:01.580 |I| ModLoader Patch: Patching address offset 8f66f8 <= EB 03 1F 2A len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset 8f66fc <= 55 00 80 52 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset 8f6730 <= 41 00 80 52 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset 8f67a4 <= 55 00 80 52 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset 8f67ac <= 4C 00 80 52 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset 8f6814 <= 48 00 80 52 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset c76fe0 <= 08 00 80 52 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset c76ff0 <= 08 00 80 52 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset cdd58c <= 15 00 00 14 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset eac370 <= 41 00 80 52 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset 19b1f84 <= 1F 20 03 D5 len=4
00:00:01.580 |I| ModLoader Patch: Patching address offset 19bd9e8 <= 1F 20 03 D5 len=4
00:00:01.742 |I| Ptc Initialize: Initializing Profiled Persistent Translation Cache (enabled: True).
00:00:04.071 |I| Ptc Load: Loaded Translation Cache (size: 181280734 bytes, translated functions: 144909).
00:00:04.110 |I| Ptc Load: Loaded Profiling Info (size: 461766 bytes, profiled functions: 135137).
00:00:04.122 |I| Loader LoadNsos: Loading image 0 at 0x0000000008000000...
00:00:04.124 |I| Loader LoadNsos: Loading image 1 at 0x0000000008004000...
00:00:04.162 |I| Loader LoadNsos: Loading image 2 at 0x000000000c72b000...
00:00:04.165 |I| Loader LoadNsos: Loading image 3 at 0x000000000cdd4000...
00:00:04.175 |I| ModLoader LoadCheats: Build ids found for title 0100F2C0115B6000:
    7501ABFE55FA41CFFEB46BD619BDCBF45B9CAE3A000000000000000000000000
    9A10ED9435C06733DA597D8094D9000AB5D3EE60000000000000000000000000
    A6BB7C6FFA9673769FA74ED3D7B054191F25D29C000000000000000000000000
    B9046C31EB5D31271BE970FE732D38DF49C6AA21000000000000000000000000
00:00:04.178 |I| ModLoader ApplyRomFsMods: Applying RomFS mods for Title 0100F2C0115B6000
00:00:04.209 |I| ModLoader ApplyRomFsMods: Replaced 91 file(s) over 7 mod(s). Processing base storage...
00:00:44.362 |I| ModLoader ApplyRomFsMods: Building new RomFS...
00:00:44.574 |I| ModLoader ApplyRomFsMods: Using modded RomFS
00:00:44.577 |I| Application EnsureSaveData: Ensuring required savedata exists.
00:00:44.688 |I| Loader Start: Application Loaded: The Legend of Zelda: Tears of the Kingdom v1.1.2 [0100f2c0115b6000] [64-bit]
00:00:44.700 |I| GUI.WindowThread Hid Configure: Configured Controller ProController to Player1
00:00:44.726 |I| GUI.WindowThread Hid SetupNpad: Connected Controller ProController to Player1
00:00:48.291 |N| GUI.RenderLoop Gpu PrintGpuInformation: NVIDIA NVIDIA GeForce GTX 1070 (Vulkan v1.3.236, Driver v532.3.0.0)
00:00:48.303 |I| GPU.MainThread Gpu LoadShaders: Loading 2511 shaders from the cache...
00:01:04.796 |I| GPU.MainThread Gpu LoadShaders: Shader cache loaded.
00:01:07.694 |I| HLE.GuestThread.21 Ptc LoadTranslations: 134611 translated functions loaded
00:01:07.737 |I| HLE.GuestThread.21 Ptc MakeAndSaveTranslations: 0 of 526 functions translated | Thread count: 7
00:01:08.158 |I| HLE.GuestThread.21 Ptc MakeAndSaveTranslations: 526 of 526 functions translated | Thread count: 7 in 0.4188814 s
00:01:08.290 |S| HLE.OsThread.22 ServiceAm GetAppletResourceUserId: Stubbed. {appletResourceUserId: 1}
00:01:08.304 |S| HLE.OsThread.22 ServiceAm AcquireForegroundRights: Stubbed. 
00:01:08.304 |S| HLE.OsThread.22 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: False ; unknownFlag2: False ; unknownFlag3: True}
00:01:08.305 |S| HLE.OsThread.22 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:01:08.321 |S| HLE.OsThread.22 ServicePctl Initialize: Stubbed. 
00:01:09.458 |S| HLE.OsThread.22 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: True ; unknownFlag2: False ; unknownFlag3: True}
00:01:09.458 |S| HLE.OsThread.22 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:01:09.459 |S| HLE.OsThread.22 ServiceAm SetRestartMessageEnabled: Stubbed. {restartMessageEnabled: True}
00:01:09.459 |S| HLE.OsThread.22 ServiceAm SetOperationModeChangedNotification: Stubbed. {operationModeChangedNotification: True}
00:01:09.460 |S| HLE.OsThread.22 ServiceAm SetPerformanceModeChangedNotification: Stubbed. {performanceModeChangedNotification: True}
00:01:09.548 |S| HLE.OsThread.10 ServiceNv FinishInitialize: Stubbed. 
00:01:09.555 |I| HLE.OsThread.22 SurfaceFlinger CreateLayerFromId: Creating layer 1
00:01:09.557 |I| HLE.OsThread.22 SurfaceFlinger CreateLayerFromId: Creating layer 2
00:01:09.577 |S| HLE.OsThread.10 ServiceNv InitializeEx: Stubbed. 
00:01:09.594 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:01:09.646 |S| HLE.OsThread.10 ServiceNv SetNvMapFd: Stubbed. 
00:01:09.647 |S| HLE.OsThread.10 ServiceNv AllocGpfifoEx2: Stubbed. 
00:01:09.647 |S| HLE.OsThread.10 ServiceNv AllocObjCtx: Stubbed. 
00:01:09.648 |S| HLE.OsThread.10 ServiceNv SetErrorNotifier: Stubbed. 
00:01:09.649 |S| HLE.OsThread.10 ServiceNv SetUserData: Stubbed. 
00:01:09.649 |S| HLE.OsThread.10 ServiceNv SetTimeout: Stubbed. 
00:01:09.649 |S| HLE.OsThread.10 ServiceNv SetTimeslice: Stubbed. 
00:01:09.651 |S| HLE.OsThread.10 ServiceNv ZcullBind: Stubbed. 
00:01:09.679 |S| HLE.OsThread.22 ServiceAcc InitializeApplicationInfo: Stubbed. {TitleId: 72324500776771584}
00:01:09.682 |S| HLE.OsThread.22 ServiceAcc IsUserAccountSwitchLocked: Stubbed. 
00:01:09.682 |S| HLE.OsThread.22 ServiceNs CheckAddOnContentMountStatus: Stubbed. 
00:01:09.699 |S| HLE.OsThread.9 ServiceHid ActiveNpadImpl: Stubbed. {appletResourceUserId: 1 ; revision: 3}
00:01:09.704 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadIdType: Stubbed. 9 Players: Player1,Player2,Player3,Player4,Player5,Player6,Player7,Player8,Handheld
00:01:09.704 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 98 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight}
00:01:11.036 |S| HLE.OsThread.10 ServiceNv ZbcSetTable: Stubbed. 
00:01:11.037 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:01:13.975 |S| HLE.OsThread.9 ServiceHid ActivateMouse: Stubbed. {appletResourceUserId: 1}
00:01:13.977 |S| HLE.OsThread.9 ServiceHid ActivateKeyboard: Stubbed. {appletResourceUserId: 1}
00:01:14.238 |I| HLE.OsThread.6 AudioRenderer AcquireSessionId: Registered new renderer (0)
00:01:14.241 |I| HLE.OsThread.6 AudioRenderer Initialize: Initializing with REV11
00:01:14.273 |I| AudioProcessor.Worker AudioRenderer Work: Starting audio processor
00:01:14.274 |I| AudioRendererManager.Worker AudioRenderer SendCommands: Starting audio renderer
00:01:14.276 |I| HLE.OsThread.6 AudioRenderer Start: Starting renderer id 0
00:01:14.322 |S| HLE.OsThread.6 ServiceAudio QueryAudioDeviceOutputEvent: Stubbed. 
00:01:14.325 |S| HLE.OsThread.6 ServiceAudio GetActiveChannelCount: Stubbed. 
00:01:14.363 |I| AudioProcessor.Worker Audio EnsureAudioStreamSetup: New audio stream setup with a target sample count of 240
00:01:15.141 |S| HLE.OsThread.22 ServiceAm SetOperationModeChangedNotification: Stubbed. {operationModeChangedNotification: True}
00:01:15.141 |S| HLE.OsThread.22 ServiceAm SetPerformanceModeChangedNotification: Stubbed. {performanceModeChangedNotification: True}
00:01:15.288 |I| Ptc Save: Saved Translation Cache (size: 181759298 bytes, translated functions: 145435).
00:01:15.660 |S| HLE.OsThread.10 ServiceNv ZbcSetTable: Stubbed. 
00:01:15.663 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:01:15.664 |S| HLE.OsThread.10 ServiceNv ZbcSetTable: Stubbed. 
00:01:15.664 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:01:16.467 |S| HLE.OsThread.22 ServiceAm GetMainAppletExpectedMasterVolume: Stubbed. 
00:01:16.467 |S| HLE.OsThread.22 ServiceAm GetLibraryAppletExpectedMasterVolume: Stubbed. 
00:01:16.467 |S| HLE.OsThread.22 ServiceAm SetExpectedMasterVolume: Stubbed. 
00:01:16.727 |S| HLE.OsThread.9 ServiceHid StartSixAxisSensor: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 131075}
00:01:16.739 |S| HLE.OsThread.6 ServiceAudio GetActiveChannelCount: Stubbed. 
00:01:16.896 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: bootup_time 
 Report: {
    "BootupTimeUs": 12879,
    "system_report_tag": 2175206328
}

00:01:16.899 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: erepo_region 
 Report: {
    "Region": 1,
    "Language": 1,
    "StandardTimeName": "UTC",
    "UtcOffsetSeconds": 0,
    "system_report_tag": 2759414308
}

00:01:16.899 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: erepo_time 
 Report: {
    "ActiveTime": 8,
    "UpdatedTime": 0,
    "system_report_tag": 2759414308
}

00:01:16.900 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: erepo_playstyle 
 Report: {
    "system_report_tag": 2759414308
}

00:01:16.901 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: erepo_network_status 
 Report: {
    "NetworkInterface": {
        "TypeCode": 64,
        "Value": "0x00000000000000000000000000000000"
    },
    "LinkLevel": {
        "TypeCode": 65,
        "Value": "0x00"
    },
    "FrequencyBand": {
        "TypeCode": 66,
        "Value": "0x00000000000000000000"
    },
    "system_report_tag": 2759414308
}

00:01:16.972 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: erepo_active_beacon 
 Report: {
    "IntervalTime": 0,
    "UpdatedTime": 0,
    "system_report_tag": 0
}

00:01:19.747 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 98 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair}
00:01:19.749 |S| HLE.OsThread.9 ServiceHid DisconnectNpad: Stubbed. {appletResourceUserId: 4294967296 ; npadIdType: Player2}
00:01:19.749 |S| HLE.OsThread.9 ServiceHid DisconnectNpad: Stubbed. {appletResourceUserId: 4294967296 ; npadIdType: Player3}
00:01:19.750 |S| HLE.OsThread.9 ServiceHid DisconnectNpad: Stubbed. {appletResourceUserId: 4294967296 ; npadIdType: Player4}
00:01:28.691 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:01:29.740 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: event_player_start 
 Report: {
    "PlayTime": 3,
    "ReleaseVersion": 1334,
    "PlayDataUniqueID": 1197295449,
    "EventInfo": "Dm_OP_0037",
    "PlayerPosX": 3.406024E-11,
    "PlayerPosY": -0.15038963,
    "PlayerPosZ": 3.406211E-11,
    "system_report_tag": 2175206328
}

00:01:29.824 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: event_player_start 
 Report: {
    "PlayTime": 3,
    "ReleaseVersion": 1334,
    "PlayDataUniqueID": 1197295449,
    "EventInfo": "Dm_OP_0038",
    "PlayerPosX": 3.406024E-11,
    "PlayerPosY": -0.15038963,
    "PlayerPosZ": 3.406211E-11,
    "system_report_tag": 2175206328
}

00:01:30.037 |S| HLE.OsThread.22 ServiceAm SetIdleTimeDetectionExtension: Stubbed. {idleTimeDetectionExtension: 1}
00:01:40.385 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: option_data 
 Report: {
    "PlayTime": 3,
    "ReleaseVersion": 1334,
    "PlayDataUniqueID": 1197295449,
    "BoolOptions": 1099,
    "CameraRotateSpeed": 3,
    "DisplayMode": 0,
    "MessageWindowTransparency": 0,
    "VoiceLanguage": 1,
    "Action": 1,
    "Cancel": 2,
    "Jump": 3,
    "Attack": 4,
    "SpecialPower": 5,
    "Attention": 6,
    "Throw": 7,
    "Bow": 8,
    "PlusMenu": 9,
    "MinusMenu": 10,
    "SpecialPowerShortcut": 11,
    "CompanionShortcut": 13,
    "PouchShortcut": 14,
    "Whistle": 12,
    "Crouch": 25,
    "Scope": 33,
    "PresetType": 0,
    "system_report_tag": 2175206328
}

00:01:40.396 |S| HLE.OsThread.22 ServiceAm SetIdleTimeDetectionExtension: Stubbed. {idleTimeDetectionExtension: 0}
00:01:57.107 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:01:59.150 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:01:59.635 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: event_player_start 
 Report: {
    "PlayTime": 197178,
    "ReleaseVersion": 1334,
    "PlayDataUniqueID": 323581307,
    "EventInfo": "DefaultBootEvent",
    "PlayerPosX": 1536.1942,
    "PlayerPosY": 385.06122,
    "PlayerPosZ": -1843.8228,
    "system_report_tag": 2175206328
}

00:02:08.197 |I| .NET ThreadPool Worker Ptc Save: Saved Profiling Info (size: 461808 bytes, profiled functions: 135148).
00:02:25.465 |I| HLE.OsThread.16 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 98
 UserId: 00000000000000010000000000000000
 Room: erepo_active_beacon 
 Report: {
    "IntervalTime": 60,
    "UpdatedTime": 60,
    "system_report_tag": 0
}

00:02:38.203 |I| .NET ThreadPool Worker Ptc Save: Saved Profiling Info (size: 461806 bytes, profiled functions: 135148).
00:06:53.200 |W| HLE.GuestThread.23 KernelSvc : WaitSynchronization(handleIndex: 0xFFFFFFFF) = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.29 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.24 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.28 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.33 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.25 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.30 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.34 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.26 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.31 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.35 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.27 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.32 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.36 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.38 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.37 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.44 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.42 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.46 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.51 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.53 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.200 |W| HLE.GuestThread.43 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.57 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.45 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.50 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.52 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.61 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.54 KernelSvc : WaitSynchronization(handleIndex: 0xFFFFFFFF) = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.70 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.73 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.58 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.59 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.65 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.56 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.66 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.60 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.78 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.72 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.79 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.75 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.63 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.64 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.201 |W| HLE.GuestThread.67 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.202 |W| HLE.GuestThread.71 KernelSvc : WaitForAddress() = TerminationRequested
00:06:53.263 |I| HLE.OsThread.6 AudioRenderer Stop: Stopping renderer id 0
00:06:53.263 |I| HLE.OsThread.6 AudioRenderer Stop: Stopped renderer id 0
00:06:53.264 |I| HLE.OsThread.6 AudioRenderer ReleaseSessionId: Unregistered renderer (0)
00:06:53.269 |I| AudioProcessor.Worker AudioRenderer Work: Stopping audio processor
00:06:53.277 |W| GUI.WindowThread ServiceNv Close: Closing channel
00:06:53.306 |I| HLE.OsThread.6 AudioRenderer StopLocked: Stopped audio renderer

### OS

windows 10

### Ryujinx version

1.1.842

### Game version

1.1.2

### CPU

i7-7700

### GPU

1070

### RAM

32gb
